### ISSUE TITLE
Bug: Helm Chart bug, can not use a cascading expander with priority in it, configmap missing

### ISSUE THREAD

**Issue by AndrewFarley** [kubernetes/autoscaler#4502]
# Issue
The conditional creation logic for the [priority-expander-configmap.yaml](https://github.com/kubernetes/autoscaler/blob/master/charts/cluster-autoscaler/templates/priority-expander-configmap.yaml#L2):
```
{{- if hasKey .Values.extraArgs "expander" }}
{{- if and (.Values.expanderPriorities) (eq .Values.extraArgs.expander "priority") -}}
```

Directly conflicts with ability to pass multiple expanders to cascade, info [from the FAQ](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-are-expanders):
```
Multiple expanders may be passed, i.e. .cluster-autoscaler --expander=priority,least-waste
```

which results in it not being possible to set...
```yaml
  extraArgs:
    expander: priority,least-waste  # <--- this
  expanderPriorities: |-
    10:
      - '.*g2\..*'
```

And have it still set the `expanderPriorities` configmap.  Right now I can set this expander value above and it works properly, except the configmap is not created because of this flawed conditional logic.

# Bugfix
Possible bugfix is to change the "eq" to contains in that second line, I'm submitting an MR now.

**Comment by ddelange** [comment]
Can this issue be closed as of #4665?

**Comment by AndrewFarley** [comment]
Yep!
